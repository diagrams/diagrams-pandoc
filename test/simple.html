<p>Here is a square:</p>
<p><img src="images/3895b48f2a2f113a.png" /></p>