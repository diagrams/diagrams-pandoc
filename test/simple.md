Here is a square:

``` {.diagram}
example = square 1
```

