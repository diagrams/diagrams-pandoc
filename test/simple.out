[ Para
    [ Str "Here"
    , Space
    , Str "is"
    , Space
    , Str "a"
    , Space
    , Str "square:"
    ]
, Para
    [ Image
        ( "" , [] , [] ) [] ( "images/3895b48f2a2f113a.png" , "" )
    ]
]